libs/api/domains/license-service/src/lib/resolvers/pkPass.resolver.ts

```diff
@@ -0,0 +1,87 @@
+import type { User } from '@island.is/auth-nest-tools'
+import {
+  CurrentUser,
+  IdsUserGuard,
+  Scopes,
+  ScopesGuard,
+} from '@island.is/auth-nest-tools'
+import { ApiScope } from '@island.is/auth/scopes'
+import { Audit } from '@island.is/nest/audit'
+
+import { UseGuards } from '@nestjs/common'
+import { Args, Mutation, Resolver } from '@nestjs/graphql'
+import { GeneratePkPassInput } from '../dto/GeneratePkPass.input'
+import { GenericPkPass } from '../dto/GenericPkPass.dto'
+import { GenericPkPassQrCode } from '../dto/GenericPkPassQrCode.dto'
+import { VerifyLicenseBarcodeInput } from '../dto/VerifyLicenseBarcodeInput'
+import { VerifyLicenseBarcodeResult } from '../dto/VerifyLicenseBarcodeResult.dto'
+import { LicenseService } from '../licenseService.service'
+import { GenericPkPassVerification } from '../dto/GenericPkPassVerification.dto'
+import { VerifyPkPassInput } from '../dto/VerifyPkPass.input'
+
+@UseGuards(IdsUserGuard, ScopesGuard)
+@Resolver(() => GenericPkPass)
+@Audit({ namespace: '@island.is/api/license-service' })
+export class PkPassResolver {
+  constructor(private readonly licenseServiceService: LicenseService) {}
+
+  @Mutation(() => GenericPkPass, {
+    name: 'generatePkPass',
+  })
+  @Audit()
+  async generatePkPass(
+    @CurrentUser() user: User,
+    @Args('input') input: GeneratePkPassInput,
+  ): Promise<GenericPkPass> {
+    const pkpassUrl = await this.licenseServiceService.generatePkPassUrl(
+      user,
+      input.licenseType,
+    )
+
+    return {
+      pkpassUrl,
+    }
+  }
+
+  @Mutation(() => GenericPkPassQrCode, {
+    name: 'generatePkPassQrCode',
+  })
+  @Audit()
+  async generatePkPassQrCode(
```

Comment:
Very hard to see if something changed when you split one file up to many files 👀 😵‍💫  Tried reading it quickly over but did not see any big change so assuming it is as before for this file.

Reply:
Ah sorry, the pkpass stuff should be unchanged.